web/views/error_view.ex

```diff
@@ -9,6 +9,14 @@ defmodule Org.ErrorView do
     "Internal server error"
   end
 
+  def render("401.json", _assigns) do
+    %{error: "Please sign in"}
```

Comment:
Also, we should probably work out a standard format for responses. 

```js
{
  data: ...
  status: ...
  page?: ... if we want to page results? (probably not a big concern)
}

{
  status: 401
  message: 'Unauthorized'
}
```
❓ 

Let's discuss more in Slack when you have time.